vulture_allowlist.py

```diff
@@ -298,6 +298,7 @@
 event
 deserialize
 serialized_event
+PydanticEventSerializer
```

Comment:
Why not imported?

Reply:
It doesn't need to be imported, vulture detects it automatically.